EDITORIAL CLOSING: The XMPP Newsletter May 2026 (#1683)
xsf/xmpp.org: content/blog/newsletter/2026-06-05-newsletter.md

---
title: The XMPP Newsletter May 2026
date: 2026-06-05
categories: ['Newsletter']

> summary: "In this issue: Fully online 29th XMPP Summit announcement! XMPP at FOSSY 2026. XMPP Sprint in Berlin. EnvsBot XMPP Helper Bot for the envs pubnix - BETA, Movim's first 2026 Goal Reached! What's Next?, What’s your oldest Openfire deployment?, and more! Update your clients with aTalk, Conversations, Converse.js, Monal, Monocles, Poezio and Profanity. Update your servers: Prosody IM 13.0.6, Openfire 5.0.5, MongooseIM 6.7.0, and Specifications update: one proposed XEP and three updated ones!"

Reviewer: "Movim's first 2026 Goal Reached! What's Next?, What’s your oldest Openfire deployment?, and more!" this looks a bit mixed together. Maybe we can cross one or two elements out or join them better?

Author: "Movim's first 2026 Goal Reached! What's Next?, What’s your oldest Openfire deployment?, and more!" ->
"Movim's first 2026 Goal Reached!, What’s your oldest Openfire deployment?, and more!" 
Does that look better?

Reviewer: Yes

## XSF Announcements

### XSF Membership

The XMPP Standards Foundation (XSF) is excited to announce the [29th XMPP Summit](/2026/05/xmpp-summit-29/), the first XMPP Summit to take place [fully online](/2026/05/xmpp-summit-29/#the-online-xmpp-summit)! The XMPP Summit will be held from Friday 4th September to Saturday 5th September 2026, both days between 13:00 - 16:00 UTC. The XSF invites everyone interested in development of XMPP technologies to attend, and discuss all things XMPP remotely!

## XMPP Events

- [XMPP Sprint in Berlin](https://wiki.xmpp.org/web/Sprints/2026-06_Berlin) (DE / EN): will take place in June, from Friday 19th to Sunday 21st 2026, at the [Wikimedia Deutschland e.V.](https://www.openstreetmap.org/node/2551527703#map=19/52.498414/13.381049) offices in Berlin, Germany. If this sounds like the right event for you, come and join us! Just make sure to list yourself [here](https://wiki.xmpp.org/web/Sprints/2026-06_Berlin#Attendees), so we know how many people will attend and we can plan accordingly. If you have any questions or concerns, join us at the chatroom: [sprints@muc.xmpp.org](xmpp:sprints@muc.xmpp.org?join)!

> - XMPP at FOSSY 2026: This year's edition of [FOSSY](https://2026.fossy.ca/), the fourth _Free and Open Source Software Yearly_ conference, will take place during the month of August, from Thursday 6th to Sunday 9th 2026 at the [University of British Columbia](https://www.ubc.ca/), Vancouver, Canada. As always, there will be an [XMPP Track](https://2026.fossy.ca/pages/tracks/). The [call for proposals](https://2026.fossy.ca/call-for-proposals/) is closed as of June 2st, 2026. Once again, this year [JMP](https://jmp.chat/) was pleased to do its annual offer for funding to the potential speakers who would like to host a talk on the XMPP track.

Reviewer: ```suggestion
- XMPP at FOSSY 2026: This year's edition of [FOSSY](https://2026.fossy.ca/), the fourth _Free and Open Source Software Yearly_ conference, will take place during the month of August, from Thursday 6th to Sunday 9th 2026 at the [University of British Columbia](https://www.ubc.ca/), Vancouver, Canada. As always, there will be an [XMPP Track](https://2026.fossy.ca/pages/tracks/). The [call for proposals](https://2026.fossy.ca/call-for-proposals/) is closed as of June 2nd, 2026. Once again, this year [JMP](https://jmp.chat/) is pleased to do its annual offer for funding to the potential speakers who would like to host a talk on the XMPP track.
```

## Videos and Talks

- [Independent Messaging (XMPP) unter iOS (Monal)](https://makertube.net/w/bgLLsnJXg1vuHouM3xfqmE), by [eversten.net](https://makertube.net/a/eversten.net/video-channels) for [XMPP Tutorials DE](https://makertube.net/c/xmpp_tutorials_de/videos). [DE] ([EN],[ES] subtitles available).
- [Independent Messaging (XMPP) unter Android (Snikket)](https://makertube.net/w/6UMPCbNANq7VQSvCGq4DEd), by [eversten.net](https://makertube.net/a/eversten.net/video-channels) for [XMPP Tutorials DE](https://makertube.net/c/xmpp_tutorials_de/videos). [DE] ([EN],[ES] subtitles available).
- [Independent Messaging (XMPP) unter Android (Conversations)](https://makertube.net/w/uaVGgjiE18WTR49eDYtJGt), by [eversten.net](https://makertube.net/a/eversten.net/video-channels) for [XMPP Tutorials DE](https://makertube.net/c/xmpp_tutorials_de/videos). [DE] ([EN],[ES] subtitles available).
- [Conversations (XMPP): Feature-Preview für Version 2.20.0](https://makertube.net/w/c62E2MbfE5WSpB9uVC6p5p), by [eversten.net](https://makertube.net/a/eversten.net/video-channels) for [XMPP Tutorials DE](https://makertube.net/c/xmpp_tutorials_de/videos). [DE]

## XMPP Articles

> - [EnvsBot XMPP Helper Bot for the envs pubnix - BETA](https://dan.envs.net/blog/2026/2026-05-01-envsbot-xmpp-helper-bot-for-the-envs-pubnix---beta), by [~dan](https://dan.envs.net/about) for [~dan's website on envs.net blog](https://dan.envs.net/blog/).

Reviewer: I have no clue what this could be about. Maybe that's ok. Maybe some context should be given. What do you think?

Author: I just added a description to it. Please let me know if it is worth keeping or maybe just delete the whole entry. =)

Reviewer: Yes, looks fine

- [What’s your oldest Openfire deployment?](https://discourse.igniterealtime.org/t/what-s-your-oldest-openfire-deployment/96473), by [guus](https://discourse.igniterealtime.org/u/guus/summary) for the [Ignite Realtime community](https://discourse.igniterealtime.org/).
- [¿Qué es XMPP: mensajería federada sin depender de WhatsApp?](https://tuiter.ovh/que-es-xmpp-mensajeria-federada/), by [elenamusk](https://tuiter.ovh/author/elena_/) for the [Tuiter.Rocks blog](https://tuiter.ovh/blog/). [ES]

## XMPP Software News

### XMPP Clients and Applications

- [aTalk](https://github.com/cmeng-git/atalk-android/blob/master/README.md) has released versions [5.4.0](https://github.com/cmeng-git/atalk-android/releases/tag/5.4.0), [5.5.0](https://github.com/cmeng-git/atalk-android/releases/tag/5.5.0) and [6.0.0](https://github.com/cmeng-git/atalk-android/releases/tag/6.0.0) of its encrypted instant messaging with video call and GPS features for Android. This versions bring a whole lot of 'under the hood' changes. Please refer to the [release notes](https://github.com/cmeng-git/atalk-android/blob/master/aTalk/ReleaseNotes.txt) for all the details.

> - [Conversations](https://conversations.im/) has released versions [2.19.16](https://codeberg.org/iNPUTmice/Conversations/releases/tag/2.19.16) and [2.20.0](https://codeberg.org/iNPUTmice/Conversations/releases/tag/2.20.0) for Android. This releases brings some significant UI work, an improved media browser with ability to share and delete media files, store media internally by default (with setting to automatically save to gallery), send audio recordings immediately instead of attaching (plus setting to restore the old behavior), `Show QR code` that now includes preauth parameter on servers with easy invites, a search bar for easier access to searches from the home screen and moved the attachment button closer to input field. Make sure to take a look at the [changelog](https://codeberg.org/iNPUTmice/Conversations/src/branch/master/CHANGELOG.md) for all the details!

Reviewer: ```suggestion
- [Conversations](https://conversations.im/) has released versions [2.19.16](https://codeberg.org/iNPUTmice/Conversations/releases/tag/2.19.16) and [2.20.0](https://codeberg.org/iNPUTmice/Conversations/releases/tag/2.20.0) for Android. These releases bring some significant UI work, an improved media browser with ability to share and delete media files, store media internally by default (with setting to automatically save to gallery), send audio recordings immediately instead of attaching (plus setting to restore the old behavior), `Show QR code` that now includes preauth parameter on servers with easy invites, and a search bar for easier access to searches from the home screen. Make sure to take a look at the [changelog](https://codeberg.org/iNPUTmice/Conversations/src/branch/master/CHANGELOG.md) for all the details!
```

> {{< figure src="/images/newsletter/2026-06-05/conversations.png" caption="Conversations: A prominen search bar on the main screen and a reworked media attachment flow." >}}

Reviewer: ```suggestion
{{< figure src="/images/newsletter/2026-06-05/conversations.png" caption="Conversations: A prominent search bar on the main screen and a reworked media attachment flow." >}}
```

> - [Converse.js](https://conversejs.org) has released versions [13.0.0](https://github.com/conversejs/converse.js/releases/tag/v13.0.0) and [13.0.1](https://github.com/conversejs/converse.js/releases/tag/v13.0.1) of its open-source web-based XMPP chat client, with various additions, improvements and bugfixes. This versions add support for [message reactions](/extensions/xep-0444.html) allowing users to react to messages with emojis, [message replies](/extensions/xep-0461.html) to specific messages, autocomplete of possible XMPP servers when registering a new account, half a dozen fixes and quite some work under the hood!. Head over to the [changelog](https://github.com/conversejs/converse.js/blob/master/CHANGES.md) for the all the details!

Reviewer: ```suggestion
- [Converse.js](https://conversejs.org) has released versions [13.0.0](https://github.com/conversejs/converse.js/releases/tag/v13.0.0) and [13.0.1](https://github.com/conversejs/converse.js/releases/tag/v13.0.1) of its open-source web-based XMPP chat client, with various additions, improvements and bugfixes. These versions add support for [message reactions](/extensions/xep-0444.html) allowing users to react to messages with emojis, [message replies](/extensions/xep-0461.html) to specific messages, autocomplete of possible XMPP providers when registering a new account, half a dozen fixes and quite some work under the hood!. Head over to the [changelog](https://github.com/conversejs/converse.js/blob/master/CHANGES.md) for the all the details!
```

- [Monal](https://monal-im.org/) has released version [6.4.21](https://github.com/monal-im/Monal/releases/tag/Build_iOS_1092) for iOS and macOS. This version now shows last usage timestamp of OMEMO keys, improves location sharing accuracy and contact last activity timestamps, and it also fixes several bugs!

> - [Monocles](https://monocles.eu/more) has released versions [2.1.6](https://codeberg.org/monocles/monocles_chat/releases/tag/CodebergRelease-v2.1.6), [2.2](https://codeberg.org/monocles/monocles_chat/releases/tag/CodebergRelease-v2.2), [2.2.1](https://codeberg.org/monocles/monocles_chat/releases/tag/CodebergRelease-v2.2.1), and [2.2.2](https://codeberg.org/monocles/monocles_chat/releases/tag/v2.2.2) of its chat client for Android. This versions introduce a large amount of new features, implementations, fixes and bug fixes that is way larger than we could ever list in here. Among the more prominent highlights you'll find [ephemeral messages](/extensions/xep-0466.html), live location sharing, improved create and restore backup, use internal hidden storage by default, local database encryption, improved backup encryption & reliability, secure deletion & migration recovery, UI/UX improvements, media gallery, and a lot more! Make sure to check the links for all the details!. **Note**: due to some delays with the Play Store updates, this releases are only available through [F-droid](https://f-droid.org/en/packages/de.monocles.chat/) for the time being.

Reviewer: ```suggestion
- [Monocles](https://monocles.eu/more) has released versions [2.1.6](https://codeberg.org/monocles/monocles_chat/releases/tag/CodebergRelease-v2.1.6), [2.2](https://codeberg.org/monocles/monocles_chat/releases/tag/CodebergRelease-v2.2), [2.2.1](https://codeberg.org/monocles/monocles_chat/releases/tag/CodebergRelease-v2.2.1), and [2.2.2](https://codeberg.org/monocles/monocles_chat/releases/tag/v2.2.2) of its chat client for Android. These versions introduce a large amount of new features, implementations, and bug fixes that is way larger than we could list in here. Among the more prominent highlights you'll find [ephemeral messages](/extensions/xep-0466.html), live location sharing, improved create and restore backup, use internal hidden storage by default, local database encryption, improved backup encryption & reliability, secure deletion & migration recovery, UI/UX improvements, media gallery, and a lot more! Make sure to check the links for all the details!. **Note**: due to some delays with the Play Store updates, these releases are only available through [F-droid](https://f-droid.org/en/packages/de.monocles.chat/) for the time being.
```

> - [Poezio](https://poez.io/en/) has released versions [0.18](https://blog.mathieui.net/poezio-0-18.html) of its console client for XMPP. This new release has mostly internal improvements, with some new features like [message retraction](/extensions/xep-0424.html) as the sending side, various improvements to the `react` plugin, notably a way to convert `:emoji:` aliases to real emojis and a new way of showing or hiding groupchat presences along with a few other improvements and fixes. You can find all the details in the link to the release.

Reviewer: ```suggestion
- [Poezio](https://poez.io/en/) has released version [0.18](https://blog.mathieui.net/poezio-0-18.html) of its console client for XMPP. This new release has mostly internal improvements, with some new features like [message retraction](/extensions/xep-0424.html) at the sending side, various improvements to the `react` plugin, notably a way to convert `:emoji:` shortcodes to real emojis, and a new way of showing or hiding groupchat presences along with a few other improvements and fixes. You can find all the details in the link to the release.
```

> - [Profanity](https://profanity-im.github.io/) has released version [0.18.1](https://github.com/profanity-im/profanity/releases/tag/0.18.1) of its console based XMPP client written in C. This release brings bug fixes to connect with empty resource, resolves issues with async editor, resolves DB migration failure, and restores message receipt requests when capabilities are unknown among many other things. Please make sure to read the [changelog](https://github.com/profanity-im/profanity/blob/master/CHANGELOG) for all the details!

Reviewer: ```suggestion
- [Profanity](https://profanity-im.github.io/) has released version [0.18.1](https://github.com/profanity-im/profanity/releases/tag/0.18.1) of its console based XMPP client. This release brings bug fixes to connect with an empty resource, resolves issues with async editor, resolves DB migration failures, and restores message receipt requests when capabilities are unknown, among many other things. Please make sure to read the [changelog](https://github.com/profanity-im/profanity/blob/master/CHANGELOG) for all the details!
```

### XMPP Servers

- [MongooseIM](https://www.erlang-solutions.com/technologies/mongooseim/) has released [MongooseIM 6.7.0](https://github.com/esl/MongooseIM/releases/tag/6.7.0) with more additions, changes and fixes than what we can reasonably list in here! Make sure to read the [changelog](https://github.com/esl/MongooseIM/blob/master/CHANGELOG.md) for all the details!
- The [Ignite Realtime community](https://www.igniterealtime.org/) is happy to announce the release of [Openfire 5.0.5](https://discourse.igniterealtime.org/t/openfire-5-0-5-release/96468). The [full changelog](https://download.igniterealtime.org/openfire/docs/latest/changelog.html) has more details with the highlights being bug fixes and bundled library updates whilst they continue to work on an upcoming 5.1.0 feature release.

> - [Prosody IM](https://prosody.im/) is pleased to announce version [13.0.6](https://blog.prosody.im/prosody-13.0.6-released/), a new minor release from our stable branch. This release fixes a handful of bugs which were discovered and fixed since the 13.0.5 release. Most of these are minor, but a few of them are important fixes. Make sure to read the [changelog](https://prosody.im/doc/release/13.0.6) for all the details! 

Reviewer: ```suggestion
- [Prosody IM](https://prosody.im/) is pleased to announce version [13.0.6](https://blog.prosody.im/prosody-13.0.6-released/), a new minor release from the stable branch. This release fixes a handful of bugs which were discovered and fixed since the 13.0.5 release. Most of these are minor, but a few of them are important fixes. Make sure to read the [changelog](https://prosody.im/doc/release/13.0.6) for all the details! 
```

### XMPP Libraries & Tools

- [go-xmpp](https://github.com/xmppo/go-xmpp/blob/master/README.md) versions [0.3.3](https://github.com/xmppo/go-xmpp/releases/tag/v0.3.3) and [0.3.4](https://github.com/xmppo/go-xmpp/releases/tag/v0.3.4) have been released.

> - [go-sendxmpp](https://salsa.debian.org/mdosch/go-sendxmpp/-/blob/master/README.md?ref_type=heads), a tool to send messages to an XMPP contact or MUC inspired by [sendxmpp](https://sendxmpp.hostname.sk/), versions [0.15.6](https://salsa.debian.org/mdosch/go-sendxmpp/-/releases/v0.15.6) and [0.15.8](https://salsa.debian.org/mdosch/go-sendxmpp/-/releases/v0.15.8) have been released. Full details on the [changelog](https://salsa.debian.org/mdosch/go-sendxmpp/-/blob/master/CHANGELOG.md?ref_type=heads).

Reviewer: ```suggestion
- [go-sendxmpp](https://salsa.debian.org/mdosch/go-sendxmpp/-/blob/master/README.md?ref_type=heads), a tool to send messages to an XMPP contact or MUC inspired by [sendxmpp](https://sendxmpp.hostname.sk/), versions [0.15.6](https://salsa.debian.org/mdosch/go-sendxmpp/-/releases/v0.15.6) and [0.15.8](https://salsa.debian.org/mdosch/go-sendxmpp/-/releases/v0.15.8) have been released. Full details in the [changelog](https://salsa.debian.org/mdosch/go-sendxmpp/-/blob/master/CHANGELOG.md?ref_type=heads).
```

> - [jabber.el](https://codeberg.org/emacs-jabber/emacs-jabber/src/branch/master/README.org), the XMPP client for [Emacs](https://www.gnu.org/software/emacs/), versions [0.10.7](https://codeberg.org/emacs-jabber/emacs-jabber/releases/tag/0.10.7), [0.10.8](https://codeberg.org/emacs-jabber/emacs-jabber/releases/tag/0.10.8), [0.10.9](https://codeberg.org/emacs-jabber/emacs-jabber/releases/tag/0.10.9) and [0.10.10](https://codeberg.org/emacs-jabber/emacs-jabber/releases/tag/0.10.10) have been released. Full details on the [changelog](https://codeberg.org/emacs-jabber/emacs-jabber/src/branch/master/CHANGELOG.org).

Reviewer: ```suggestion
- [jabber.el](https://codeberg.org/emacs-jabber/emacs-jabber/src/branch/master/README.org), the XMPP client for [Emacs](https://www.gnu.org/software/emacs/), versions [0.10.7](https://codeberg.org/emacs-jabber/emacs-jabber/releases/tag/0.10.7), [0.10.8](https://codeberg.org/emacs-jabber/emacs-jabber/releases/tag/0.10.8), [0.10.9](https://codeberg.org/emacs-jabber/emacs-jabber/releases/tag/0.10.9) and [0.10.10](https://codeberg.org/emacs-jabber/emacs-jabber/releases/tag/0.10.10) have been released. Full details in the [changelog](https://codeberg.org/emacs-jabber/emacs-jabber/src/branch/master/CHANGELOG.org).
```

- [picomemo](https://github.com/mierenhoop/picomemo/blob/master/README.md), a compact and portable implementation of the cryptography required for XMPP's OMEMO (E2EE), version [1.2.0](https://github.com/mierenhoop/picomemo/releases/tag/1.2.0) has been released. Read the details in the [changelog](https://github.com/mierenhoop/picomemo/compare/1.1.0...1.2.0).
- [slidgnal](https://codeberg.org/slidge/slidgnal/src/branch/main/README.md), a [feature-rich](https://slidge.im/docs/slidgnal/main/user/features.html) [Signal](https://signal.org/) to XMPP puppeteering [gateway](/extensions/xep-0100.html), based on [slidge](https://slidge.im/) and [signalmeow](https://github.com/mautrix/signal/tree/main/pkg/signalmeow), version [0.3.0beta11](https://codeberg.org/slidge/slidgnal/releases/tag/v0.3.0beta11) has been released. You can read the intermediate changelog from [0.3.0beta10 to 0.3.0beta11](https://codeberg.org/slidge/slidgnal/compare/v0.3.0beta10...v0.3.0beta11) for all the details.

> - [Slixmpp](https://codeberg.org/poezio/slixmpp/src/branch/master/README.rst), the MIT licensed XMPP library for Python 3.7+ version [1.15.0](https://codeberg.org/poezio/slixmpp/releases/tag/slix-1.15.0) has been released. You can read the official [release announcement](https://blog.mathieui.net/slixmpp-1-15.html) for all the details.

Reviewer: ```suggestion
- [slixmpp](https://codeberg.org/poezio/slixmpp/src/branch/master/README.rst), the MIT licensed XMPP library for Python 3.7+ version [1.15.0](https://codeberg.org/poezio/slixmpp/releases/tag/slix-1.15.0) has been released. You can read the official [release announcement](https://blog.mathieui.net/slixmpp-1-15.html) for all the details.
```
